en/docs/administer/managing-users-and-roles/managing-permissions.md

```diff
@@ -157,7 +157,13 @@ Additionally, all permissions listed under <strong>Admin/Monitor</strong> in the
 </tbody>
 </table>
 
-## Adding role-based permissions
+## Adding scope-based permissions
+
+WSO2 API-Manager (APIM) publisher, dev portal, and admin portal UIs work on top of APIM-specific Rest APIs. Access to these APIs is given based on OAuth token scopes. When a user logs in to the publisher or dev portal, the access token is generated with a specific set of scopes based on the user's roles.
+
+By default, Internal/creator, Internal/publisher, Internal/subscriber are mapped to these specific scopes. As a result, any user with these roles can access publisher and devportal features. 
+
+You could customize these scope-to-role mapping and grant permission to access these UIs to different user roles. For example, if you have a user with role X and want to grant permission to access dev portal, you could assign all devportal-related scopes to that role. The following section describes how to do this.
```

Comment:
By default, Internal/creator, Internal/publisher, and Internal/subscriber roles map to these specific scopes. As a result, any user with these roles can access API Publisher and Developer Portal features. 
			
It is possible to customize these scope-to-role mapping and grant permission to access these UIs to different user roles. 
i.e., If you have a user with role X and want to grant permission to access Developer Portal, you could assign all Developer Portal related scopes to that role. The following section describes how to do this.